front/src/components/login_forms.js

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import {
+    Grid,
+    Paper,
+    TextField,
+    FormControlLabel,
+    Checkbox,
+    Button,
+    Link,
+    Typography
+} from '@mui/material';
+
+const paperStyle = {
+    padding: 20,
+    height: "45vh",
+    minHeight: 350,
+    width: 450,
+    margin: "0"
+}
+
+export const StudentLogin = () => {
+    const buttonStyle = {
+        margin: "8px 0"
+    }
+    return (
+        <Grid>
+            <Paper elevation={0} style={paperStyle}>
+                <Grid align="center">
+                    <h2>Rejoindre une session</h2>
+                </Grid>
+                <Grid>
+                    <TextField
+                        id="standard-basic"
+                        label="Room id"
+                        placeholder="Entrer l'id de la room"
+                        variant="standard"
+                        fullWidth
+                        style={buttonStyle}
+                    />
+                    <TextField
+                        id="standard-basic"
+                        label="Pseudo"
+                        placeholder="Entrer un Pseudo"
+                        variant="standard"
+                        fullWidth
+                        style={buttonStyle}
+                    />
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                name="checkedB"
+                                color="primary"
+                            />
+                        }
+                        label="Se souvenir de moi"
```

Comment:
Does this button is necessary for the student? I mean, the id room will change everytime, and he may want to change its username too.